src/Api/Search.php

```diff
@@ -25,6 +25,7 @@ class Search
                 'code',
                 'name',
                 'description',
+                'teaserDescription',
```

Comment:
I think we need to check the learning path short description as well because it is different. Not sure about that!

Reply:
we have already checked me and @jadkhateradmin , it does not have a short description

Reply:
@omaya-adm The description of the LPs is already a short description, according to my understanding of the TMS. the elaborate descriptions are on courses.